wiki/Rules/Code_of_conduct_for_modding_and_mapping/en.md

```diff
@@ -4,58 +4,33 @@ The **code of conduct for modding and mapping** is a set of rules and guidelines
-  - **Subjective Issues** - Something that you believe or think on a personal level. Might become intersubjective if many people agree with it.
-- **Unless the concept behind a beatmap is fundamentally flawed from the start**, modding should aim to improve the map **in its current design** - not force your own style upon it. If you truly believe the map has too many significant issues to address individually, try to give a general statement of why this is the case and what direction you think the map should go in to get back on track.
-- When pointing out "issues", inspect each thoroughly and consider exactly what it is about the particular "thing" you dislike. Then, try to offer a potential solution or improvement and consider exactly how that suggestion will improve the map. If necessary, try multiple suggestions to see which work best — the more help you can offer the mapper, the better.
-
-Suggestions themselves should ideally consist of four main parts, though based on the experience of the mapper and/or your personal relationship to them, you may want to be more lenient:
-
-1. **A statement of the issue itself and where it can be found** - including a timestamp is a huge help so that the mapper can find the part in question quickly.
-2. **A brief explanation of why this causes a problem** - this is essential in convincing a mapper to even consider your suggestion. Mappers are much more receptive to criticism and change when it's accompanied by clear, understandable and valid reasoning.
-3. **One or more alternative suggestions, accompanied by explanations of how they address the problem(s)** - much like the above, explanation is key here. Even if you can convince the mapper that their current work can be improved, what makes your suggestion better than theirs, or anyone else's?
-4. **The wording (or even language) used should take the mapper into consideration.** Are they a fluent English speaker? If not, stick to simple and easy-to-understand language. If you both speak the same language, by all means write in that language, but try to append an English summary of your mods and mod replies if possible, to ensure everyone stays on the same page. If necessary, support your suggestions with screenshots, illustrations, or even short video clips to help get your point across.
-
-There are any number of ways to present your mod post, so feel free to experiment, but try to keep the visuals simple and formatted in a way that it is easy for the mapper and others to work with.
+- **The goal of modding is to help a mapper improve their work.** If you're not interested in helping people, steer clear of the discussion.
+- **Consider what you're posting, why you're posting it, and who you're posting it to.** This will ensure your best intentions are conveyed and your post is easy to understand.
```

Comment:
Hmm, imo putting it more directly and bluntly is exactly what we want. We want users, especially those less experienced, to know at face value what the conduct expectation should be. Putting it in a more roundabout way as suggested may not get the intended effect and leave users equally confused.

Reply:
i think it's quite the opposite honestly, i don't see how it's more roundabout to specify _why_ it's good to consider these things and i don't think any information was lost

Reply:
Ultimately I'm indifferent, so I'll let others chime in. Though the current wording seemed to get the point across more directly (Consider X, Y, and Z. This achieves X, Y, and Z), that is why I felt the proposed wording felt less direct for what is basically achieving the same thing.

Reply:
I disagree with cyndifusic's proposal because the more clear wording works better as an explanation. cyndifusic's wording is harder to understand IMO, which goes against the standard that wiki pages shouldn't be hard to read.

Reply:
Will resolve for the time being then, keeping the blunter wording. Feel free to reopen if anyone thinks the opposite.